experimental/GaloisGrp/Group.jl

```diff
@@ -145,12 +145,11 @@ Determines representatives `g` for all right-cosets of `G` modulo `H`
   such that `H^g` contains the element `s`.
 """
 function short_right_transversal(G::PermGroup, H::PermGroup, s)
-  C = GAP.Globals.ConjugacyClasses(H.X)
+  C = conjugacy_classes(H)
   cs = GAP.Globals.CycleStructurePerm(s.X)
   can = []
-  for i=1:length(C)
-    c = C[i]
-    r = GAP.Globals.Representative(c)
+  for c in C
+    r = representative(c)
     if cs == GAP.Globals.CycleStructurePerm(r)
```

Comment:
Now `C` is an array of Oscar classes, `c` is such a class, `r` is an Oscar permutation, thus we should write `GAP.Globals.CycleStructurePerm(r.X)`.

Reply:
Indeed. I rewrote the rest of the function now, too